nix/nixos/cardano-graphql-service.nix

```diff
@@ -4,15 +4,6 @@
 let
   cfg = config.services.cardano-graphql;
   sources = import ../sources.nix;
```

Comment:
@disassembler @craigem 
Can you please review the changes to the nixos service? This is no longer a pre-start operation, so we just need `hasura` to be on the path.